GrandCentralBoard/Widgets/Bonus/BonusModels.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  Created by Krzysztof Werys on 24/02/16.
+//  Copyright © 2016 Krzysztof Werys. All rights reserved.
+//
+
+import Foundation
+import Decodable
+import GrandCentralBoardCore
+
+struct BubbleImage {
+    let localImageName: String?
+    let url: String?
+    
+    var remoteImage: UIImage?
+    var localImage: UIImage? {
```

Comment:
local image is a fallback image, right? Maybe we should name it `fallbackImage`?